cvmfs/network/download.h

```diff
@@ -22,15 +22,18 @@
 #include "crypto/hash.h"
 #include "dns.h"
 #include "duplex_curl.h"
+#include "health_check.h"
 #include "jobinfo.h"
 #include "network_errors.h"
+#include "sharding_policy.h"
```

Comment:
```suggestion
#include "network/sharding_policy.h"
```
(plus reorder alphabetically)